cardano-cli/src/Cardano/CLI/Types/Errors/TxCmdError.hs

```diff
@@ -217,6 +222,10 @@ renderTxCmdError = \case
     prettyError e
   TxCmdFeeEstimationError e ->
     prettyError e
+  TxCmdPoolMetadataHashError url ->
+    "Hash of the pool metadata file is not valid. Url:" <+> pretty (L.urlToText url)
+  TxCmdHashCheckError url e ->
+    "Hash of the file is not valid. Url:" <+> pretty (L.urlToText url) <+> pretty (displayException e)
```

Comment:
```suggestion
    "Hash of the file is not valid. Url:" <+> pretty (L.urlToText url) <+> prettyException e
```

Reply:
Nice, addressed in: https://github.com/IntersectMBO/cardano-cli/pull/951/commits/8031868408d59a47b0834033fecf372875a0d65e